syntax = "proto3";

package akcommon;

enum ChannelType{
  ADC_4_20_CHANNEL_TYPE = 0;
  ADC_0_5_CHANNEL_TYPE = 1;
  PCNT_CHANNEL_TYPE = 2;
  VBAT_CHANNEL_TYPE = 3;
  TEMP_CHANNEL_TYPE = 4;
  HEALTH_COUNTERS_CHANNEL_TYPE = 5;
  GSM_SIGNAL_CHANNEL_TYPE = 6;
  GPS_CHANNEL_TYPE = 7;
  BLOB_CHANNEL_TYPE = 8;
  GPIO_CHANNEL_TYPE = 9;
  MODBUS_UINT32_CHANNEL_TYPE = 10;
  MODBUS_SINT32_CHANNEL_TYPE = 11;
  MODBUS_F32_CHANNEL_TYPE = 12;
  ADC_S32_CHANNEL_TYPE = 13;
  TOUCH_CHANNEL_TYPE = 14;
  LORA_SIGNAL_CHANNEL_TYPE = 15;
}
